app/database.js

```diff
@@ -56,8 +61,9 @@ define(['sqlite', 'app/config', 'jquery'], function (sqlite, config) {
 	// Function for Handeling query Error
 	function onError(db, error) {
 		console.log("this error " + error.message);
+		alert('Something went wrong while excuting your request\n please try again! ')
```

Comment:
As you are using internationalisation you should make it possible for the application to translate this string. As it stands now, no matter the selected language the alerts in English.